src/index.ts

```diff
@@ -274,16 +274,20 @@ function launchMetals(
           "Name: ",
           ""
         ]);
-        const arg: MetalsNewScalaFileParams = {
-          directory: parentDir,
-          name: fileName,
-          kind: desiredFileType.kind
-        };
-        const result = await client.sendRequest(ExecuteCommandRequest.type, {
-          command: "new-scala-file",
-          arguments: [arg]
-        });
-        workspace.openResource(result);
+        if (fileName === "") {
```

Comment:
I think it would make sense to trim the file name before checking, unless we want to admit blank file names

Reply:
_quickly merges because how could I mess this up_

Yes, you're correct 😁